doc/object_properties.adoc

```diff
@@ -160,19 +162,20 @@ When calculating the texture color, add this much lighting. Maximum value is `mi
 Text displayed about an object, empty by default for all objects except players, in which case, it is their name.
 
 === `nametag_color`
+* field type: `color_def` or `false`
 * internal type: `ColorSpec`
 
 The color that the nametag text will be.
 
-TIP: This property can be set to `false` to disable it.
+TIP: This property can be set to `false`(bool) to disable it.
```

Comment:
Unnecessary, `false` is always a bool.

Reply:
https://github.com/minetest/minetest_docs/pull/12#discussion_r778418292 ...

Reply:
> [#12 (comment)](https://github.com/minetest/minetest_docs/pull/12#discussion_r778418292) ...

Strings should IMO be quoted.